htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxViewHandlerInterceptor.java

```diff
@@ -118,7 +118,7 @@ private void addHxHeaders(HtmxResponse htmxResponse, HttpServletResponse respons
             response.setHeader(HtmxResponseHeader.HX_RETARGET.getValue(), htmxResponse.getRetarget());
         }
         if (htmxResponse.getReswap() != null) {
-            response.setHeader(HtmxResponseHeader.HX_RESWAP.getValue(), htmxResponse.getReswap().getValue());
+            response.setHeader(HtmxResponseHeader.HX_RESWAP.getValue(), htmxResponse.getReswap().toString());
```

Comment:
I would add a different method to convert the `HtmxReswap` object to the string interpretation that htmx uses in the header. Maybe something like `asHeaderValue()`. Normally `toString()` is for debugging values and we should be free to change that at any time I think.

Reply:
Ok.